internal/envoy/v3/listener.go

```diff
@@ -277,6 +277,14 @@ func (b *httpConnectionManagerBuilder) DefaultFilters() *httpConnectionManagerBu
 				),
 			},
 		},
+		&http.HttpFilter{
+			Name: "envoy.filters.http.lua",
+			ConfigType: &http.HttpFilter_TypedConfig{
+				TypedConfig: protobuf.MustMarshalAny(&lua.Lua{
+					InlineCode: "-- Placeholder for per-Route or per-Cluster overrides.",
+				}),
+			},
+		},
```

Comment:
Are there any issues/conflicts with this + the lua filter added by `FilterMisdirectedRequests`?

Reply:
i *think* this test should prove it doesnt: https://github.com/projectcontour/contour/blob/d344821cf1e02eb1414432c2a81afd95348df3ce/test/e2e/httpproxy/cookie_rewrite_test.go#L632-L640